SocketCAN/rxServers.py

```diff
@@ -0,0 +1,114 @@
+import socket
+import uuid
+from multiprocessing import Process
+import struct
+import sys
+
+SERVER_IP = "127.0.0.1"
+try:
+	SERVER_IP = socket.gethostbyname(socket.gethostname())
+except Exception as e:
+	print("Could not get this machine's IP address, defaulting to 127.0.0.1")
+
+# The basic CAN frame structure and the sockaddr structure are defined
+#   in include/linux/can.h:
+#     struct can_frame {
+#             canid_t can_id;  /* 32 bit CAN_ID + EFF/RTR/ERR flags */
+#             __u8    can_dlc; /* frame payload length in byte (0 .. 8) */
+#             __u8    __pad;   /* padding */
+#             __u8    __res0;  /* reserved / padding */
+#             __u8    __res1;  /* reserved / padding */
+#             __u8    data[8] __attribute__((aligned(8)));
+#     };   
+
+# To match this data structure, the following struct format can be used:
+can_frame_format = "<LL8s"
+# Unsigned Long Integer (little endian), unsigned char (byte), three pad bytes, eight chars (bytes)
```

Comment:
This comment does not match the frame format.

Reply:
No, it does not. The second L contains DLC and Micros, which is done on lines 76/77. This was done because there were no format characters from https://docs.python.org/2/library/struct.html with length of 3. Do you suggest this be changed to "<LB3s8s" and the lines 76/77 logic removed?

Reply:
You need to update the comments to match the code.